sdks/python/src/opik/cli/import_command.py

```diff
@@ -19,16 +18,16 @@
 
 console = Console()
 
+# Constants for import command
+DEFAULT_PROJECT_NAME = "default"
```

Comment:
not used
btw default project name is "Default Project" (if I understood correctly what the "project" means here :) )